keyboards/keebio/levinson/keymaps/issmirnov/rgb.c

```diff
@@ -0,0 +1,66 @@
+#include "rgb.h"
+
+// TODO gate this debugging header
+#include <print.h>
+
+// Wired up in layer_state_set_user in keymap.c
+uint32_t layer_state_set_rgb(uint32_t state) {
+  static uint8_t old_layer = 255;
+  uint8_t new_layer = biton32(state);
+  if (old_layer != new_layer) {
```

Comment:
I may be missing something, but I don't see why you'd need to check this. 

This function is called every time the layers change. There are some circumstances that may call it with no effective change, but there is no real benefit for doing this

Reply:
This is probably an artifact from some earlier experiments. I think this was an optimization I found in another keymap, but since neither you nor I know why this is needed I'm removing it. Thank you!